src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -317,16 +317,19 @@
             collapsed?     (and initial-joined? (:joined community))
             overlay-shown? (boolean (:sheets (rf/sub [:bottom-sheet])))]
         [scroll-page/scroll-page
-         {:cover-image                    cover
-          :collapsed?                     collapsed?
-          :logo                           logo
-          :page-nav-right-section-buttons (page-nav-right-section-buttons id)
-          :name                           name
-          :on-scroll                      #(reset! scroll-height %)
-          :navigate-back?                 true
-          :background-color               (colors/theme-colors colors/white colors/neutral-95)
-          :height                         148
-          :overlay-shown?                 overlay-shown?}
+         {:cover-image      cover
+          :collapsed?       collapsed?
+          :logo             logo
+          :name             name
+          :on-scroll        #(reset! scroll-height %)
+          :navigate-back?   true
+          :background-color (colors/theme-colors colors/white colors/neutral-95)
```

Comment:
is it possible to add the `theme` param here 🙏

Reply:
I commented usage of`:background-color` in `scroll-page` on both - community overview and discover pages and don't see any visual changes.
So I think we can get rid of background-color at all here. Transparent scroll-page is good enough) 
In case we will need some coloring later, it probably would be more idiomatic to create a separate component through `with-theme`, because now `scroll-page` has no access to theme.